template/build/core/phing/tasks/deploy.xml

```diff
@@ -83,6 +83,8 @@
   <target name="deploy:commit">
     <!-- We make these commands quiet because they can cause the output to be so long that Travis CI stops logging. -->
     <exec command="git add -A" dir="${deploy.dir}" logoutput="true" passthru="true" checkreturn="true"/>
+    <!-- We are forcing vendor everywhere to add here in case a global git ignore flags it -->
+    <exec command="git add *vendor/* -f" dir="${deploy.dir}" logoutput="true" passthru="true" checkreturn="true"/>
```

Comment:
Well... The wildcard before could open some other issues despite what it fixes.  Not realistic issues... I could name a directory maliciousdirectoryvendor/* and it will be added.  Not realistic, but possible.  Doing "vendor/*" ensures it's docroot/vendor/* only.